requirements.txt

```diff
@@ -1,7 +1,9 @@
-lxml>=3.4.4
+lxml=3.6.2
```

Comment:
There's a regression in the lxml that pulls in a realtime library which works on linux, but deep-sixes the build on OS X, will check if that's been resolved